src/connectors/refinement-list/__tests__/connectRefinementList-test.js

```diff
@@ -130,10 +129,10 @@ describe('connectRefinementList', () => {
     });
 
     // test that rendering has been called during init with isFirstRendering = false
-    expect(rendering.callCount).toBe(2);
-    expect(rendering.lastCall.args[1]).toBe(false);
+    expect(rendering.mock.calls.length).toBe(2);
```

Comment:
You could use the shorthand method: `.toHaveBeenCalledTimes(2)`

Reply:
Ok gotcha! Thanks!